src/Compilers/CSharp/Portable/Binder/Semantics/BestTypeInferrer.cs

```diff
@@ -196,7 +186,7 @@ private static TypeSymbol GetBestType(ArrayBuilder<TypeSymbol> types, Conversion
                     {
                         best = null;
                     }
-                    else
+                    else if ((object)better != (object)best)
```

Comment:
>((object)better != (object)best) [](start = 28, length = 32)

Is this necessary? Seems like the only difference with whether we update `bestIndex` (ie. keep the first or the last)